konsave

```diff
@@ -10,38 +10,73 @@
 import os, shutil, argparse
 from zipfile import ZipFile, ZIP_DEFLATED, is_zipfile
 
+try:
+    import yaml
+    import bitches
```

Comment:
I'm assuming you forgot to delete this line.

Reply:
Oh my fucking god. I'm really sorry, I did this in order to test if the catch works.

As this is my first pull request, how can I resolve this?